storage/google_drive/drive_client_wrapper.py

```diff
@@ -195,6 +195,12 @@ def _create_file(source_file_path, target_folder_id, target_file_name=None):
              f"'{source_file_path}' - done. File id is '{file.get('id')}'")
 
 
+def _delete_file(file_id):
+    log.info(f"Deleting file '{file_id}'...")
```

Comment:
```suggestion
    """
    Permanently delete a file, skipping the trash.
    """
    log.info(f"Deleting file '{file_id}'...")
```

Reply:
as a warning incase someone else wants to call the function

Reply:
Thanks, switched to a warning and added your description. The note about skipping the trash is helpful, I didn't think to check that.